flopy/modpath/mp7particledata.py

```diff
@@ -967,7 +985,14 @@ def get_cell_release_points(subdivisiondata, cellid, extent) -> Iterator[tuple]:
 
 
 def get_release_points(
-    subdivisiondata, grid, k=None, i=None, j=None, nn=None, localz=False
+    subdivisiondata,
+    grid,
+    k=None,
+    i=None,
+    j=None,
+    nn=None,
+    localz=False,
```

Comment:
kicking myself for the inconsistency. eventually we'll make it `local_z` on the mf6 side and follow here I guess.